packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bugsnag/core",
-  "main": "src/index.js",
+  "main": "./dist/index.cjs",
```

Comment:
Forgot to update this in previous PR but it didn't matter because it seems only react-native uses it and before this PR the react-native notifier wasn't doing any imports from `@bugsnag/core`, only direct to source ones